frappe_io/www/docs/user/en/guides/integration/webhooks.md

```diff
@@ -25,6 +25,12 @@ To add a Webhook go to,
 - If your request structure is based on forms, you can select fields from the document in the table, which uses the fieldname as the `key`.
 - If your request structure is based on JSON, you can insert fields from the document using jinja templating (make sure to wrap your fields with double-quotes).
 
+### Webhook Security
+
```

Comment:
@Alchez Add

> \> Introduced in Version 13

Reply:
@surajshetty3416, I've added the reference